worker/src/main.rs

```diff
@@ -206,32 +166,16 @@ fn main() {
         return;
     }
     if let Some(_matches) = matches.subcommand_matches("init-shard") {
-        match _matches.values_of("shard") {
-            Some(values) => {
-                for shard in values {
```

Comment:
Was this intentionally multiple shards?

In the scripts it was not used with multiple shards. As I have also started with python wrappers for the worker, I suggest we handle multiple shards in the wrapper and serve a consistent CLI with the rust worker.

Reply:
it was intentional. but so far not important I guess